modules/ossm-release-2-5-5.adoc

```diff
@@ -0,0 +1,34 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-2-5-5_{context}"]
+= {SMProductName} version 2.5.5
+
+This release of {SMProductName} is included with the {SMProductName} Operator 2.6.2, addresses Common Vulnerabilities and Exposures (CVEs), and is supported on {product-title} 4.14 and later.
+
+
+[id=ossm-release-2-5-5-components_{context}]
+== Component updates
+
+|===
+|Component |Version
+
+|Istio
+|1.18.7
+
+|Envoy Proxy
+|1.26.8
+
+|Kiali Server
+|1.73.15
+|===
+
+[id="ossm-fixed-issues-2-5-5_{context}"]
+== Fixed issues
+
+* https://issues.redhat.com/browse/OSSM-8001[OSSM-8001] Previously, when `runAsUser` and `runAsGroup` were set to the same value in pods, the proxy GID was incorrectly set to match the container's GID, causing traffic interception issues with iptables rules applied by Istio CNI. Now, containers can have the same value for runAsUser and runAsGroup, and iptables rules apply correctly.
```

Comment:
Can `runAsUser` and `runAsGroup` have nouns behind the names? Maybe "parameters"?
Do we need to define GID on first appearance here?

```suggestion
* https://issues.redhat.com/browse/OSSM-8001[OSSM-8001] Previously, when the `runAsUser` and `runAsGroup` parameters were set to the same value in pods, the proxy GID was incorrectly set to match the container's GID, causing traffic interception issues with iptables rules applied by Istio CNI. Now, containers can have the same value for the `runAsUser` and `runAsGroup` parameters, and iptables rules apply correctly.
```